### ISSUE TITLE
park_start higher than park_end and out of bounds in last step

### ISSUE THREAD

**Issue by grafvontrips** [rl-institut/simbev#24]
In rare cases park_start in the last step is higher than park_end and also outside of the simulated range. 
So far I could not find a cause for this behaviour. Across all my test runs, it seems to occur with about 1/250 chance. It seems like it always affects the same simulated vehicles, for example `bev_luxury_00050_standing_times.csv` or `bev_medium_00198_standing_times.csv` always seems to be affected.
See the added csv for an example.

[bev_luxury_00050_standing_times.txt](https://github.com/rl-institut/simbev/files/6508365/bev_luxury_00050_standing_times.txt)

The following config has been used:

```
# This file is part of eDisGo and simbev
# here the simulation parameters for simbev are set

[basic]
# basic parameters for the simulation
# ===========================
# definition of regional type to use for simulating driving behaviour
# efficiency of charging points
# timesteps (should stay at 15 min, best results)
# number of days to simulate, max: 14, suggested: 14 for best results
# minimum soc allowed before fast charging event is triggered

regio_type = LR_Klein
eta_cp =  1
stepsize = 15
days = 14
soc_min = 0.2

[rampup_ev]

# ramp up of electric vehicles
# ===========================
# number of cars for each type that are simulated
BEV_mini = 400
BEV_medium = 400
BEV_luxury = 200
PHEV_mini = 0
PHEV_medium = 0
PHEV_luxury = 0

[tech_data_cc_slow]

# charging capacity of each car type for AC-charging
# ===========================
# max. charging capacity in kW

BEV_mini = 11
BEV_medium = 22
BEV_luxury = 50
PHEV_mini = 3.7
PHEV_medium = 11
PHEV_luxury = 11

[tech_data_cc_fast]

# charging capacity of each car type for DC-charging
# ===========================
# max. charging capacity in kW
# PHEV are not capable of DC-charging

BEV_mini = 50
BEV_medium = 50
BEV_luxury = 150
PHEV_mini = 0
PHEV_medium = 0
PHEV_luxury = 0

[tech_data_bc]

# battery capacity of each car type
# ===========================
# battery capacity in kWh

BEV_mini = 30
BEV_medium = 65
BEV_luxury = 90
PHEV_mini = 20
PHEV_medium = 20
PHEV_luxury = 20

[tech_data_ec]

# energy consumption of each car type
# ===========================
# average energy consumption in kWh/km

BEV_mini = 0.13
BEV_medium = 0.16
BEV_luxury = 0.2
PHEV_mini = 0.16
PHEV_medium = 0.16
PHEV_luxury = 0.16

[charging_probabilities]

# charging probabilities for all locations
# ===========================
#

slow = EV_charging_point_probability.csv
fast = EV_fast_charging_probability.csv


[sim_params]

# simulation parameters
# ===========================
# parameters for running the simulation on your pc

num_threads = 1
seed = 3
```

**Comment by khelfen** [comment]
encountering the same issue

**Comment by Mosc5** [comment]
This is solved in PR #43
